### ISSUE TITLE
[question] migrating legacy tunnels to named tunnels

### ISSUE THREAD

**Issue by ahatdude** [aeleos/cloudflared#28]
Hello,
I have been having some problems with cloudflared as of late.
It could be some misconfiguration but before I delete and restart it, I was looking around and I found this:
https://developers.cloudflare.com/cloudflare-one/connections/connect-apps/do-more-with-tunnels/migrate-legacy-tunnels/

Does this change any of the steps when installing cloudflared on unRAID?

Thanks for your time.

**Comment by aeleos** [comment]
Yes, this tutorial is only for legacy tunnels. For the current tunnels, all you need to do is go through the official CF guide and modify the template run command such that you start the container with your key rather than a configuration file. Then you can create an tunnels and routes through the CF panel.

Example command:
```
tunnel run --token ${CF_TUNNEL_TOKEN}
```
